src/components/Explore/TokenRow.tsx

```diff
@@ -0,0 +1,314 @@
+  justify-content: center;
+  align-items: center;
+  transform: scale(1.2);
+`
+
+const SwapContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 16px 4px;
+  gap: 10px;
+`
+
+const SwapButton = styled.button`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 8px;
+  gap: 6px;
+  font-weight: 600;
+
+  width: 54px;
+  height: 32px;
+
+  background: ${({ theme }) => theme.primary2};
+  border-radius: 12px;
+  border: none;
+  color: ${({ theme }) => theme.white};
+  &:hover {
+    background-color: ${({ theme }) => darken(0.05, theme.primary2)};
+  }
+`
+
+const TokenSymbol = styled.span`
+  color: ${({ theme }) => theme.text3};
+`
+const ArrowContainer = styled.div`
+  padding: 0px 0px 0px 4px;
+  display: flex;
+  flex-direction: column;
+`
+
+const SortingCategory = styled.span`
+  color: ${({ theme }) => theme.primary1};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+
+const SortArrowContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 0px 2px 0px 0px;
+`
+
+function getHeaderCategory(category: string, sortCategory: string, sortDecreasing: boolean) {
+  if (sortCategory === category) {
+    return (
+      <SortingCategory>
+        <SortArrowContainer>{sortDecreasing ? <ArrowDown size={14} /> : <ArrowUp size={14} />}</SortArrowContainer>
+        {category}
+      </SortingCategory>
+    )
+  }
+  return category
+}
+
+export function headerRow() {
+  const possibleSortCategories = ['Market Cap', 'Price', '% Change']
+  const sortCategory = possibleSortCategories[0]
+  const sortDecreasing = true
+  return (
+    <HeaderRow>
+      <div className="col-hide-4"></div>
+      <div></div>
+      <Trans>
+        <NameContainer>Name</NameContainer>
+        <PriceContainer>{getHeaderCategory('Price', sortCategory, sortDecreasing)}</PriceContainer>
+        <PercentChangeContainer>{getHeaderCategory('% Change', sortCategory, sortDecreasing)}</PercentChangeContainer>
+        <MarketCapContainer className="col-hide-3">
+          {getHeaderCategory('Market Cap', sortCategory, sortDecreasing)}
+        </MarketCapContainer>
+        <VolumeContainer className="col-hide-2">1D Volume</VolumeContainer>
+      </Trans>
+    </HeaderRow>
+  )
+}
+
+export default function TokenRow({
+  key,
+  tokenAddress,
+  data,
+  listNumber,
+  timePeriod,
+}: {
+  key: string
+  tokenAddress: string
+  data: TokenData
+  listNumber: number
+  timePeriod: TimePeriod
+}) {
+  const token = useToken(tokenAddress)
+  const tokenName = token?.name
+  const tokenSymbol = token?.symbol
+  const tokenData = data[tokenAddress]
+  // TODO: remove magic number colors
+  // TODO: write favorited hook
+  const favorited = true
+  return (
+    <TokenRowWrapper key={key}>
+      <FavoriteContainer className="col-hide-4">
```

Comment:
i think you can remove these classNames, and then do the display: none in the FavoriteCointainer styled component instead?

Reply:
same for the other hide stuff